renderapi/transform/leaf/thin_plate_spline.py

```diff
@@ -3,9 +3,21 @@
 from renderapi.utils import encodeBase64, decodeBase64
 from .transform import Transform
 import scipy.spatial
+import logging
+import sys
 __all__ = ['ThinPlateSplineTransform']
 
 
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+
+class AdaptiveMeshEstimationError(EstimationError):
+    def __init__(self, value, transform):
```

Comment:
This is a tricky thing to do in python.  I'm pretty sure that trying to raise something like this in e.g. a multiprocessing pool will cause hanging since the reduction function pickle uses will not treat it nicely.  There is a way around this, but correct behavior should be tested if we go this route (which I think is okay).